.github/workflows/base-ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     container: baoproject/bao:latest
     steps:
       - uses: actions/checkout@v2
-      - run: doc8 source/ --ignore D000
+      - run: make format
```

Comment:
There is already a CI rule called `format` for formatting C files using clang-format. We need another name. Didn't you have issues when adding it to *ci.mk*?

Reply:
Fixed. [e134cc0](https://github.com/bao-project/bao-docs/pull/68/commits/e134cc098c330665e4212c86fc15398ae78bf61f)